Update outbreaks description (#7)
R4EPI/R4EPIs-website: content/outbreaks/_index.en.md

1. **Cholera/Acute watery diarrhea**
2. **Meningitis**
3. **Measles/Rubella**
4. **Acute Jaundice Syndrome (often suspected to be Hepatitis E)**

Some recent examples of such outbreak responses and the data analysis that was done during those outbreaks is available here:

* [Cholera epidemic in Yemen, 2016–18: an analysis of surveillance data](https://www.thelancet.com/journals/langlo/article/PIIS2214-109X(18)30230-4/fulltext)
* [Invasive Meningococcal Meningitis Serogroup C Outbreak in Northwest Nigeria, 2015 - Third Consecutive Outbreak of a New Strain](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958021/?report=reader)
* [Lessons and Challenges for Measles Control from Unexpected Large Outbreak, Malawi](https://wwwnc.cdc.gov/eid/article/19/2/12-0301_article)
* [A large outbreak of Hepatitis E virus genotype 1 infection in an urban setting in Chad likely linked to household level transmission factors, 2016-2017](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0188240)

You can access these by selecting them from the RStudio RMarkdown template menu:

> > RStudio>File>New File>R Markdownn>Template>Select Outbreak Report Template

Reviewer: I suggest to use this syntax. We can add an image later.

```suggestion
> 1. in RStudio, click on File > New File > R Markdown
> 2. select "From Template" and then select the outbreak report template you want to use
```

# Getting started with outbreak templates in R

> This section (INSERT LINK TO THE WALK THROUGH ON OUTBREAK DATA ANALYSIS) offers you a step by step guide on how to use the outbreak templates to fit your data and run the analysis.

Reviewer: I suggest adding an explicit "TODO" so that we don't forget it.

```suggestion
This section (TODO: INSERT LINK TO THE WALK THROUGH ON OUTBREAK DATA ANALYSIS) offers you a step by step guide on how to use the outbreak templates to fit your data and run the analysis.
```

4. Analysis of **personal characteristics**: demographics, signs and symptoms, attack rates, risk groups etc.
5. Analysis of **temporal characteristics**: creating graphs that display different indicators in time
6. Analysis of **spatial/geographical characteristics**: creating maps

# Other uses of the templates

As the coding included in the outbreak templates is structured in terms of time, place and person, they can also be used to help guide you through any other analysis for epidemiological data from either:

* outpatient registers
* hospital registers
* other surveillance data

You will not be able to use the same set up of the data dictionaries, but you should be able to work out how to recode and rename your variables, create graphs and tables of relevant variables and if necessary generate maps.

There is also coding to be able to guide you on conducting simple statistical comparisons between groups (chi square, t-test etc.), as well as the calculation of Odds Ratios (ORs) and Risk Ratios (RR) and running regression models for multivariate analyses.

> > **Below here is all the stuff that was written before that we need to make sure is included in the outbreak template walkthrough**

Reviewer: Again, adding a TODO so that it's easier to search for these tasks.

```suggestion
> **TODO: Below here is all the stuff that was written before that we need to make sure is included in the outbreak template walkthrough**
```
